.github/workflows/release.yaml

```diff
@@ -4,14 +4,18 @@ on:
   workflow_dispatch:
     inputs:
       version:
-        description: "The version of the template we want to release. For example 0.75.0-rc.0"
+        description: "The version of the react-native we want to release. For example 0.75.0-rc.0"
         required: true
         type: string
       is_latest_on_npm:
         description: "Whether we want to tag this template release as `latest` on NPM"
         required: true
         type: boolean
         default: false
+      dry_run:
+        description: "Run without making persistent changes to git or npm"
+        type: boolean
+        default: false
```

Comment:
I'd make the default to true for safety. Less chances to forget to tick it when we test.

Reply:
Sure can do, originally I defaulted to this because I didn't want to have to land other changes in react-native.  All things considered, that's actually a bit silly so I'll flip this.